apps/native/app/src/screens/document-detail/document-communications.tsx

```diff
@@ -0,0 +1,283 @@
+          setShowFirstReplyInfo(showFirstReplyInfo)
+        }
+
+        handleRefresh()
+      },
+    }
+
+    showModal(ComponentRegistry.DocumentReplyScreen, {
+      passProps,
+    })
+  }
+
+  useEffect(() => {
+    if (!isSkeleton && comments.length > 0) {
+      setShouldScroll(true)
+    }
+  }, [isSkeleton, comments.length])
+
+  const keyExtractor = useCallback(
+    (item: FlatListItem, index: number) => item.id ?? `comment-${index}`,
+    [],
+  )
+
+  const handleContentSizeChange = () => {
+    if (shouldScroll) {
+      setShouldScroll(false)
+      setTimeout(() => {
+        flatListRef.current?.scrollToOffset({
+          offset: 75 * (comments.length - 1),
+          animated: true,
+        })
+      }, TOGGLE_ANIMATION_DURATION + 100)
+    }
+  }
+
+  const renderItem = useCallback(
+    ({ item, index }: ListRenderItemInfo<FlatListItem>) => {
+      if (item.__typename === 'Skeleton') {
+        return <ListItemSkeleton key={item.id} showDate={false} />
+      }
+
+      return (
+        <DocumentListItem
+          key={item.id}
+          isOpen={index === comments.length - 1}
+          closeable={comments.length > 1}
+          sender={item.author ?? ''}
+          title={item.author ?? ''}
+          body={item.body ?? undefined}
+          date={item.createdDate ? formatDate(item.createdDate) : undefined}
+          hasTopBorder={index !== 0}
+        />
+      )
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [comments.length],
+  )
+
+  const data = useMemo(
+    () => (isSkeleton ? createSkeletonArr(8) : comments),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [isSkeleton, docRes.data],
+  ) as FlatListItem[]
+
+  return (
+    <>
+      <CaseNumberWrapper>
+        <Typography variant="eyebrow" color={theme.color.purple400}>
+          {intl.formatMessage({ id: 'documentCommunications.caseNumber' })}
+        </Typography>
+        <Typography variant="body3">
+          #{document?.ticket?.id ?? ticketId}
+        </Typography>
+      </CaseNumberWrapper>
+      {(!showFirstReplyInfo || !replyable) && (
+        <TopLine scrollY={scrollY} offsetTop={31} />
+      )}
+      <ListWrapper>
+        {document?.closedForMoreReplies && (
+          <Container>
+            <Alert
+              type="info"
+              message={intl.formatMessage({
+                id: 'documentCommunications.cannotReply',
+              })}
+              hasBorder
+            />
+          </Container>
+        )}
+        {showFirstReplyInfo && replyable && (
+          <Container>
+            <Alert
+              type="info"
+              message={intl.formatMessage(
+                {
+                  id: 'documentCommunications.initialReply',
+                },
+                {
+                  email: user?.email ?? '',
+                },
+              )}
+              hasBorder
+            />
+          </Container>
+        )}
```

Comment:
Looks like these appear above the FlatList, but in the design these alerts are floating in the bottom near the CTA. Am I missing something?